src/agent/rustjail/src/container.rs

```diff
@@ -799,8 +799,8 @@ impl BaseContainer for LinuxContainer {
         fcntl::fcntl(pwfd, FcntlArg::F_SETFD(FdFlag::FD_CLOEXEC))?;
 
         defer!({
-            unistd::close(prfd);
-            unistd::close(pwfd);
+            let _prfd_closed = unistd::close(prfd);
+            let _pwfd_closed = unistd::close(pwfd);
```

Comment:
Could just make these `let _ =` which is briefer and probably makes it clearer that you're just throwing away the result.

Reply:
what about `unistd::close(prfd).ok()`?

Reply:
That works too.